Update LocalServer.php

ksiv opened the pull request:
Fix http 500 error caused by an empty header while using file_get_contents()

Naktibalda commented on src/Codeception/Coverage/Subscriber/LocalServer.php:
```diff
@@ -33,7 +33,7 @@ class LocalServer extends SuiteSubscriber
         'http' => [
             'method' => "GET",
-            'header' => ''
+            'header' => '"User-Agent:Codeception/2.2\r\n"'
```
Quotes don't look right.

ksiv commented:
concerning q 2. Removed version 
concerning q 1. it's not clear about constant. This is modification of a predefined array. there was 'method' predefined and I added 'header'. It's not important what to place into Header, the idea is to place something because it does not work with some modern servers without any header.  Take a look at web search "file_get_contents 500"

ksiv commented:
No positive resolution for more than a month.
